bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/i18n/somfytahoma.properties

```diff
@@ -133,9 +166,25 @@ channel-type.somfytahoma.derogation_heating_mode.description = Derogation heatin
 channel-type.somfytahoma.derogation_heating_mode.command.option.auto = Program
 channel-type.somfytahoma.derogation_heating_mode.command.option.away = Away
 channel-type.somfytahoma.derogation_heating_mode.command.option.comfort = Home
-channel-type.somfytahoma.derogation_heating_mode.command.option.frost\ protection = Frost protection
+channel-type.somfytahoma.derogation_heating_mode.command.option.frost protection = Frost protection
```

Comment:
Please restore the backslash, this is a bug in the tool.

Reply:
fixed